### PR TITLE
update accumulo from 1.6.5 to 1.8.1

### PR DESCRIPTION
**Pull request by 6487927a**
Addresses #242

### CODE REVIEW

File: mini-accumulo-cluster/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
     <properties>
         <!-- Using older version as there were problems running the 1.7.2 mini accumulo cluster -->
-        <accumulo.version>1.6.5</accumulo.version>
+        <accumulo.version>1.8.1</accumulo.version>
```
Comment (p013570) [inline mini-accumulo-cluster/pom.xml]: This entire properties block could actually now be removed, as the accumulo.version property is set to 1.8.1 in a parent pom.

**Comment by p013570** [comment]
@6487927a Would you be able to sign the license and make the minor change?

**Comment by p013570** [comment]
@6487927a - thanks very much for making the change, looks good.

I can't really merge the pull request in until you have signed our [Contributors License Agreement](https://github.com/gchq/Gaffer/wiki/GCHQ-OSS-Contributor-License-Agreement-V1.0). Are you able to do that?

**Comment by 6487927a** [comment]
@p013570 No worries at all.

I sent the agreement email on Monday shortly after your comment.  I've sent another with more information just in case.

**Comment by p013570** [comment]
@6487927a - sorry to be a pain, but the email doesn't seem to be coming through. Just to confirm, this the email template:

Email address: oss@gchq.gov.uk
Subject: GCHQ OSS Contributor License Agreement
Body: 
```
Full name:
Email:
GitHub ID:
Date:
```

**Comment by 6487927a** [comment]
No worries, I understand - sorry _I'm_ the one being the pain!  That's the address I've sent it to both times along with the supplied Subject and Body.  

If this is too much of a hassle, I have no problems with you just making the minor changes and pulling them in yourself rather than accepting this.

**Comment by p013570** [comment]
@6487927a - no, we do really appreciate you making the effort to spot the bug and fix it for us!

I'm rather confused as to where your email is disappearing to. I've tested the mailbox a couple of times and it seems to be working perfectly. It must be being filtered somewhere. It would be nice to attribute the changes to you and accept your pull request if possible.

**Comment by p013570** [comment]
Merged in via: https://github.com/gchq/gaffer-tools/pull/350

**Pull request closed without merging**